code/modules/mining/laborcamp/laborstacker.dm

```diff
@@ -20,6 +22,16 @@
 	Radio.listening = FALSE
 	locate_stacking_machine()
 
+	if(!GLOB.labor_sheet_values.len)
+		var/sheet_list = list()
+		for(var/sheet_type in subtypesof(/obj/item/stack/sheet))
+			var/obj/item/stack/sheet/sheet = sheet_type
+			if(!initial(sheet.point_value))
+				continue
+			sheet_list[initial(sheet.name)] = initial(sheet.point_value)

```

Comment:
This seems to be the cleanest way to avoid duplicate entries (e.g. due to sheet/metal and sheet/metal/fifty)

Reply:
/stack should have a `merge_type` var